test/utils/webdriver.js

```diff
@@ -19,9 +19,24 @@
  */
 const phantomjs = require('phantomjs-prebuilt');
 const webdriver = require('selenium-webdriver');
+const chrome = require('selenium-webdriver/chrome');
+const chromedriver = require('chromedriver');
+
 const By = webdriver.By;
 const until = webdriver.until;
 
+chrome.setDefaultService(new chrome.ServiceBuilder(chromedriver.path).build());
+
+let o = new chrome.Options();
+// o.addArguments('start-fullscreen');
+o.addArguments('disable-infobars');
+// o.addArguments('headless'); // running test on visual chrome browser
+o.setUserPreferences({ credential_enable_service: false });
+// const driver = new webdriver.Builder()
+//         .setChromeOptions(o)
+//         .forBrowser('chrome')
+//         .build();
+
 const driver = new webdriver.Builder()
```

Comment:
To run tests in chrome just comment this variable definition and uncomment the on above.